public/app/plugins/datasource/alertmanager/plugin.json

```diff
@@ -33,13 +42,27 @@
     },
     {
       "method": "GET",
+      "path": "alertmanager/api/v2/alerts/groups",
+      "reqRole": "Viewer",
+      "reqAction": "alert.instances.external:read"
+    },
+    {
+      "method": "GET",
+      "path": "alertmanager/api/v2/alerts",
+      "reqRole": "Viewer",
+      "reqAction": "alert.instances.external:read"
+    },
+    {
+      "method": "POST",
```

Comment:
please move POST above wildcard check
```
    {
      "method": "POST",
      "reqRole": "Editor"
    },
```
because I think it will catch the check first